docs/source/configuration.mdx

```diff
@@ -273,33 +273,45 @@ graph:
+  # the Router.
   collector: https://127.0.0.1:50051
-  # Where should we listen IF we have spawned an internal Spaceport?
+
+  # For an internal spaceport instance (when `external` is `false`), what
+  # interface and port should we listen on.
   listener: 0.0.0.0:50051
 ```
 
-#### Internal Spaceport
+#### Internal spaceport
+
+This is the default appropriate for most cirumstances, including many production deployments.  This is the default collector mode and requires no additional configuration, so long as the `graph` configuration is set with a `reference` and `key`.
+
+In this mode it is possible to further configure the address at which the internal spaceport is listening via the `listener` property.  Configuring a `listener` should only be necessary if there is a clash on the default port that Router chooses (e.g., if running multiple routers or other applications using the same port on the same host), of if it's desirable to change the bind address.
 
-This is the default and is most appropriate for evaluations, demos, development support and generally getting started. This requires no configuration by default and will be automatically started if the graph configuration is present.
-In this mode it is possible to further configure the address at which the internal spaceport is listening and the address of the spaceport to use. This should only be necessary if there is a clash on the default port, for instance if running multiple routers on the same system or clashing with another application.
+#### External spaceport
 
-#### External Spaceport
+Running an external spaceport _is not necessary in most circumstances_.  It may be desireable in certain production environments where configuration of sensitive key data or allocation of reporting resources needs to be operated centrally.  Under heavier workloads, it can also be beneficial to externalize the trace processing to reduce the amount of work that individiual Router instances take on.  The `listener` parameter is ignored for an external spaceport.
 
-In this case, external is true, the router will not start an internal spaceport task but will attempt to connect to an external process at the specified listener address. This mode of operation is intended for production environments where, for example, configuration of sensitive key data or allocation of reporting resources, can be centralised and tightly controlled. The listener parameter is ignored for an external spaceport. The collector parameter specifies the address of the shared spaceport.
+To enable the external spaceport, another Router can be run to act as the collector with the exclusive role of processing Studio data.  To configure this, the "collector" should have `external` set to `false` and an appropriate `listener` address and port combination.  Routers exporting Studio data should have `external` set to `true`, `listener` unspecified, and their `collector` property configured to point to the collector Router.
```

Comment:
I hadn't really thought about the notion of running a spaceport in a separate router, although of course that is possible.

We do have a separate binary [spaceport](https://github.com/apollographql/router/blob/main/apollo-spaceport/src/spaceport.rs) which has a minimalist CLI and is intended to be run when a standalone collector is required. The thinking here isn't fully baked, because we don't even include spaceport (the binary) in our releases yet...

Reply:
Yeah, that's a really good callout and good to capture here; I considered that briefly when writing this and went with what I suggested mostly for packaging and convenience purposes in the near-term.

I think we can pull this back out again in the future, and this PR + this conversation acts as a good artifact for now!